porcelain/miner.go

```diff
@@ -397,3 +397,105 @@ func MinerGetPeerID(ctx context.Context, plumbing mgpidAPI, minerAddr address.Ad
 	}
 	return pid, nil
 }
+
+// MinerProvingPeriod contains a miners proving period start and end as well
+// as a set of their commitments.
+type MinerProvingPeriod struct {
+	Start types.BlockHeight
+	End   types.BlockHeight
+	Set   map[string]types.Commitments
```

Comment:
Since there are two possible things this could be, I think we should be explicit and call this `ProvingSet`. Say "proving set" instead of "commitments" in the comments too.